include/ck/tensor_operation/gpu/device/device_binary_elementwise.hpp

```diff
@@ -16,7 +16,10 @@ template <typename ADataType,
           typename ComputeDataType,
           typename ElementwiseFunctor,
           index_t Dim,
-          index_t ScalarPerVector>
+          index_t M0PerThread,
```

Comment:
I'm wondering why call the dimension ```M0``` instead of just ```M```?

Reply:
I only support [M,N] before. So I used M0.
Now I can use M directly
221146a0d0019dad0f0641678948ec3763ceddf7